.coverage.MBP-Yuri.local.29469.853742

```diff
@@ -0,0 +1 @@
,[195,-193],[14,15],[-27,27],[317,-306],[-159,160],[206,-201],[229,232],[496,497],[515,524],[-114,115],[239,240],[74,75],[343,344],[62,-56],[390,391],[-469,471],[657,662],[651,652],[559,562],[109,-71],[82,83],[7,8],[585,588],[197,201],[106,107],[157,-131],[66,-1],[102,103],[8,9],[524,528],[29,-27],[420,427],[577,578],[500,506],[-1,1],[187,193],[500,501],[538,541],[316,317],[692,693],[691,692],[-39,40],[219,220],[119,121],[347,352],[224,225],[73,74],[-197,199],[208,218],[111,114],[392,-389],[75,76],[-131,133],[476,477],[9,10],[694,697],[90,91],[333,334],[67,68],[337,341],[588,589],[-306,308],[629,632],[220,221],[639,645],[330,332],[-218,219],[30,31],[80,111],[27,37],[-409,412],[95,96],[284,-218],[528,529],[153,154],[542,545],[328,329],[536,583],[531,532],[221,223],[264,266],[656,636],[160,166],[413,418],[120,-117],[551,552],[81,82],[92,95],[225,227],[485,486],[695,701],[319,347],[201,208],[514,515],[358,359],[119,120],[-111,112],[646,647],[264,265],[308,311],[57,62],[694,695],[565,566],[687,688],[314,317],[21,22],[284,221],[403,407],[647,648],[645,649],[10,11],[290,291],[594,-583],[645,646],[502,506],[-117,118],[136,137],[477,478],[-69,69],[562,565],[224,285],[337,338],[27,29],[1,2],[633,634],[39,-37],[608,621],[77,78],[124,131],[702,703],[193,197],[417,413],[495,496],[65,66],[22,24],[357,358],[46,-39],[34,-29],[510,513],[265,266],[519,524],[227,228],[78,79],[352,362],[44,45],[251,264],[314,315],[354,357],[663,664],[-583,585],[576,577],[342,345],[121,122],[339,340],[69,71],[152,153],[20,21],[475,476],[623,626],[2,3],[395,409],[635,636],[581,-536]],"/Users/apple/apps/aiosmtpd/aiosmtpd/__init__.py":[[-1,1],[1,-1]],"/Users/apple/apps/aiosmtpd/aiosmtpd/handlers.py":[[145,-142],[-68,69],[211,-206],[-140,140],[170,171],[63,66],[210,211],[61,62],[78,81],[17,20],[40,41],[-206,208],[193,195],[162,163],[203,206],[22,23],[48,51],[69,72],[-180,181],[184,189],[166,168],[-27,30],[52,53],[23,24],[24,27],[144,146],[42,44],[-7,7],[84,85],[82,83],[-149,149],[7,9],[163,164],[210,212],[75,77],[152,-151],[81,82],[81,83],[206,-193],[140,142],[42,43],[144,145],[200,203],[149,151],[44,45],[169,170],[38,56],[194,-7],[189,-178],[179,193],[171,172],[45,52],[30,-27],[174,-149],[168,169],[164,169],[14,16],[58,-56],[181,182],[72,73],[74,75],[208,210],[154,159],[13,14],[43,52],[163,166],[142,-140],[-195,196],[66,58],[62,63],[85,-68],[-193,193],[209,-206],[36,-35],[-178,178],[-88,88],[10,11],[151,154],[197,198],[146,-142],[88,90],[-142,144],[73,77],[60,61],[212,-206],[68,-33],[56,68],[-151,152],[52,54],[193,194],[185,186],[33,88],[27,33],[186,187],[12,13],[60,63],[77,78],[196,197],[-56,57],[88,140],[-35,36],[159,174],[-33,33],[140,149],[16,17],[33,35],[172,-159],[44,46],[198,-195],[47,52],[182,-180],[208,209],[57,58],[58,60],[49,52],[-159,162],[9,10],[-38,40],[90,94],[178,179],[46,47],[116,-88],[187,-184],[53,-38],[83,84],[65,66],[11,12],[46,48],[48,49],[41,42],[180,184],[21,22],[54,-38],[-184,185],[195,200],[20,21],[35,38],[94,116],[73,74],[51,52],[149,178],[178,180],[63,65]]}}
```

Comment:
Ugh ... `.coverage.*.*` files need to be excluded from commit.

Merge/rebase the latest master; I've modified `.gitignore` to exclude these files.